reference/employer/openapi.yaml

```diff
@@ -619,11 +619,12 @@ paths:
                                     type: string
                                     nullable: true
                                   value:
-                                    type:
-                                      - number
-                                      - string
-                                      - boolean
-                                    nullable: true
+                                    oneOf:
```

Comment:
lets leave this in the current format for now. unfortunately "oneOf" renders in a confusing way on stoplight :/